### PR TITLE
More cathub tests from Max

### PR DESCRIPTION
**Pull request by kirstenwinther**

### CODE REVIEW

File: catkit/hub/organize.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def collect_structures(foldername, options):
                 filetype = ase.io.formats.filetype(posix_filename)
             except Exception as e:
                 continue
-            if filetype:
+            if filetype == 'traj':
```
Comment (kirstenwinther) [inline catkit/hub/organize.py]: I couldn't make the test work for a general atoms format. The function tries to read everything in the folder.

**Comment by kirstenwinther** [comment]
I had to make a few changes to make the test pass. @mhoffman can you check that everything is ok?

File: catkit/hub/tests/make_test_slabs.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+import ase
+import ase.build
+import ase.lattice
+import ase.io
+import ase.lattice.surface
+import ase.calculators.singlepoint
+
+path = os.path.abspath(os.path.join(os.path.dirname(__file__))) + '/unorganized/'
+
```
Comment (kirstenwinther) [inline catkit/hub/tests/make_test_slabs.py]: I'm now writing the structures to the folder unorganized - wasn't that the intention?

File: catkit/hub/organize.py (hunk 1 above)
Comment (mhoffman) [reply catkit/hub/organize.py]: I had a more graceful handling of read error before. I put it back in, not sure how it disappeared. Maybe an unlucky merge somewhere. I think it is a crucial feature that we read arbitrary file as long as ASE can read them and not only *.traj file. Let's see if the tests pass now.

File: catkit/hub/tests/make_test_slabs.py (hunk 2 above)
Comment (mhoffman) [reply catkit/hub/tests/make_test_slabs.py]: Yeah, I think the test makes more sense if it writes to the unorganized folder. I couldn't get the tests to run locally and it took me too long to figure out the correct absolute path on Travis. But great if it works like this.

**Pull request merged**